docker-compose.yml

```diff
@@ -0,0 +1,151 @@
+version: '2.1'
+
+networks:
+  monitor-net:
+    driver: bridge
+
+volumes:
+  prometheus_data: {}
+  grafana_data: {}
+  caddy_data: {}
+  grafana-storage: {}
+
+services:
+  downloader:
+    container_name: dashboard_downloader
+    image: alpine:latest
+    env_file:
+      - '.env'
+    command: sh -c "apk add --no-cache curl && curl -o /tmp/slinky_dashboard.json ${DASHBOARD_URL}"
+    volumes:
+      - ./monitoring/grafana/provisioning/dashboards:/tmp  # Mount host directory to store downloaded file
+    networks:
+      - monitor-net
+
+  prometheus:
+    image: prom/prometheus:v2.48.0
+    container_name: prometheus
+    volumes:
+      - ./monitoring/prometheus:/etc/prometheus
+      - prometheus_data:/prometheus
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+      - '--storage.tsdb.path=/prometheus'
+      - '--web.console.libraries=/etc/prometheus/console_libraries'
+      - '--web.console.templates=/etc/prometheus/consoles'
+      - '--storage.tsdb.retention.time=200h'
+      - '--web.enable-lifecycle'
+    restart: unless-stopped
+    expose:
+      - 9090
+    networks:
+      - monitor-net
+    labels:
+      org.label-schema.group: "monitoring"
+
+  alertmanager:
+    image: prom/alertmanager:v0.24.0
+    container_name: alertmanager
+    volumes:
+      - ./monitoring/alertmanager:/etc/alertmanager
+    command:
+      - '--config.file=/etc/alertmanager/config.yml'
+      - '--storage.path=/alertmanager'
+    restart: unless-stopped
+    expose:
+      - 9093
+    networks:
+      - monitor-net
+    labels:
+      org.label-schema.group: "monitoring"
+
+  slinky:
+    image: ghcr.io/skip-mev/slinky-sidecar:v0.4.1
+    container_name: slinky
+    volumes:
+      - ./monitoring/slinky:/etc/slinky
+    entrypoint: [
+      "slinky", 
+      "--oracle-config-path", "/etc/slinky/oracle.json", 
+      # "--market-config-path", "/oracle/market.json", 
```

Comment:
are these comments supposed to be here @nivasan1 ?

Reply:
No, IMO we should template these parameters so I can change the environment based on arguments?